src/guardian/keys-validation/keys-validation.module.ts

```diff
@@ -0,0 +1,11 @@
+import { Module } from '@nestjs/common';
+import { KeyValidatorModule } from '@lido-nestjs/key-validation';
+import { KeysValidationService } from './keys-validation.service';
+import { LidoModule } from 'contracts/lido';
+
+@Module({
+  imports: [LidoModule, KeyValidatorModule.forFeature({ multithreaded: true })],
+  providers: [KeysValidationService],
+  exports: [KeysValidationService],
+})
+export class KeysValidationModule {}
```

Comment:
Is it used anywhere?

Reply:
of course, it will be used in the next pr

Reply:
https://github.com/lidofinance/lido-council-daemon/pull/157/files#diff-18df0bff55a3c98bcfac3b9a92dc3e2b6b8eea00defe9b2f86e90f265539da1a